### PR TITLE
Added scenarios and step definitions for the feature file

### PR DESCRIPTION
**Pull request by karunak21**
AS-2552
Added Scenarios and step definitions to the feature file.

### CODE REVIEW

File: e2e-tests/package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   },
   "devDependencies": {
     "axe-core": "^4.3.2",
+    "cucumber": "^7.0.0-rc.0",
```
Comment (shilpabasu22) [inline e2e-tests/package.json:28]: we should be using cypress-cucumber-preprocessor

**Review by shilpabasu22 — APPROVED** [review]
LGTM

**Review by karunak21 — COMMENTED** [review]
All comments have been addressed

**Pull request merged**